nav2_bt_navigator/src/bt_navigator.cpp

```diff
@@ -45,8 +45,10 @@ BtNavigator::on_configure(const rclcpp_lifecycle::State & /*state*/)
   RCLCPP_INFO(get_logger(), "Configuring");
   auto node = shared_from_this();
 
+  auto options =
+      rclcpp::NodeOptions().arguments({"__node:=bt_navigator_client_node"});
   // Support for handling the topic-based goal pose from rviz
-  client_node_ = std::make_shared<rclcpp::Node>("bt_navigator_client_node");
+  client_node_ = std::make_shared<rclcpp::Node>("_", options);
```

Comment:
Is this a leading underscore that will cause the node name to be hidden (when using command line tool to display nodes, for example)?

Reply:
No, that string is almost completely ignored. (it must, however, be a legal node name or it will raise an exception)